src/request-schemas/production/post-production.ts

```diff
@@ -9,14 +9,28 @@ export const postProduction = {
       },
       raw_materials: {
         type:"object",
+      },
+      target: {
+        type: "number",
+      },
+      buffer: {
+        type: "number",
       }
     },
-    required:["finished_item_guid", "raw_materials"]
+    required:["finished_item_guid", "raw_materials", "target", "buffer"]
   },
   response: {
     200: {
       description: 'OK',
       type: 'object',
     },
   }
+}
+
+export type POSTProductionParams = {
+  raw_materials: object;
+  finished_item_guid: string;
+  target: number;
+  buffer: number;
+  production_number?: number;
```

Comment:
It's not part of the schema? I do like the idea where user can also give the production_name